### PR TITLE
Better setup docs

### LINKED ISSUE #9: Simplify development environment configuration
It should be possible to set up the development environment with a single command—currently you have to manually create an `.env` file etc.

### PR DESCRIPTION
**Pull request by outoftime**
Mostly just adds all useful information to the [README](https://github.com/ScriptEdcurriculum/code-nation-platform/blob/af55c4530a4ede7c8d8986ce9d800b46d3897064/README.md).

Closes #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,91 +16,43 @@ Once Docker is installed, run this to bootstrap the containers:
...
-Finally, you will need to add your Firebase credentials to the local
+You will need to add your Firebase credentials to the local
 project configuration. If you have not created a Firebase app to use for
 development, [do that now](https://console.firebase.google.com/).
 
 Then run this:
 
 ```sh
-$ script/firebase-setup <YOUR-FIREBASE-APP-NAME>
+$ script/firebase-setup
```
Comment (pwjablonski) [inline README.md:31]: Does this take you through some instructions in the command line? If so maybe some mention of that.

File: README.md [hunk 2]
```diff
@@ -16,91 +16,43 @@ Once Docker is installed, run this to bootstrap the containers:
...
+REACT_APP_BUGSNAG_API_KEY=14385c1ebd16bc84362e6ce4e3f14e92
+```
 
-### Analyzing the Bundle Size
+Aside from the entries above, which you can use as-is, you will need the following:
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size
+* `REACT_APP_FIREBASE_API_KEY`: You can get this from the [Firebase console](https://console.firebase.google.com/), or by running `d/yarn.functions firebase setup:web`
+* `REACT_APP_FIREBASE_PROJECT_ID` is the ID of the Firebase project you created earlier.
+* `REACT_APP_GOOGLE_CLIENT_ID` can be found at the [Google API Console](https://console.cloud.google.com/apis/credentials); there should be an entry under **OAuth 2.0 client IDs**, but if there isn’t, you can create one. Make sure that the client ID has `http://localhost:3000` in its **Authorized JavaScript origins**; if it doesn’t, you may need to create a new client ID and add that to the authorized origins when creating it.
```
Comment (pwjablonski) [inline README.md:45]: This was by far the most confusing step I encountered when doing it the first time. Though I think it is as clear as you are going to get.

File: README.md [hunk 3]
```diff
@@ -16,91 +16,43 @@ Once Docker is installed, run this to bootstrap the containers:
...
-
-```sh
-$ docker-compose run --rm app yarn
-$ docker-compose run --rm functions yarn --cwd functions
-```
-
-Finally, you will need to add your Firebase credentials to the local
+You will need to add your Firebase credentials to the local
 project configuration. If you have not created a Firebase app to use for
 development, [do that now](https://console.firebase.google.com/).
```
Comment (pwjablonski) [inline README.md:21]: A couple of things I ran into the first time - I'm not sure if they are addressed here. 
1. Enabling google auth on firebase console
2. Creating the firebase firestore database
3. Making sure that the permissions on the database are appropriate. Should it be set to test?

File: README.md [hunk 4]
```diff
@@ -16,91 +16,43 @@ Once Docker is installed, run this to bootstrap the containers:
...
+REACT_APP_MASTER_CURRICULUM_FOLDER_ID=1wEfbo0L404VKNFpiOtOyQvllfiTs7CnO
+REACT_APP_BUGSNAG_API_KEY=14385c1ebd16bc84362e6ce4e3f14e92
+```
 
-### Analyzing the Bundle Size
+Aside from the entries above, which you can use as-is, you will need the following:
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size
+* `REACT_APP_FIREBASE_API_KEY`: You can get this from the [Firebase console](https://console.firebase.google.com/), or by running `d/yarn.functions firebase setup:web`
+* `REACT_APP_FIREBASE_PROJECT_ID` is the ID of the Firebase project you created earlier.
```
Comment (pwjablonski) [inline README.md]: Perhaps another mention of where exactly to find this

**Review by pwjablonski — COMMENTED** [review]
Looks good! I haven't had time to rerun through all these steps again but have a couple of comments based on running through similar steps the first time around.

File: README.md (hunk 2 above)
Comment (outoftime) [reply README.md:45]: :+1:

File: README.md (hunk 3 above)
Comment (outoftime) [reply README.md:21]: Good call—was thinking too narrowly about what you need to do to get set up within the repo.

File: README.md (hunk 4 above)
Comment (outoftime) [reply README.md]: OK

**Review by pwjablonski — COMMENTED** [review]
Hey Mat

Just did surface skim but looks good to me!
Doesn't seem like a blocker but may be nice to add instructions around the bin stubs in the future

Peter

**Pull request merged**